test/test_events.py

```diff
@@ -17,6 +18,26 @@ def fn_img_cleared():
 
         assert demo.config["dependencies"][0]["trigger"] == "clear"
 
+    def test_event_data(self):
```

Comment:
I would test the other Selectable components too, maybe not in this way (since launching Interfaces is slow) but at least make sure the events are added to the config successfully

Reply:
it's not a very useful to add more tests without the frontend, as the backend does nothing different per component.